src/constants.tsx

```diff
@@ -1,70 +1,87 @@
-import moment from 'moment-timezone';
-import {CellType, CommentType, Review_Stages, CellStatus} from './components/TimeCardPage/types'; 
+import moment from "moment-timezone";
+import {
+  CellType,
+  CommentType,
+  Review_Stages,
+  CellStatus,
+} from "./components/TimeCardPage/types";
```

Comment:
🤔 It probably makes more sense to store these constants at a higher level and then pass them down to TimeCardPage

Reply:
Agreed, is there a reason we would want to store inner type files? I can't think of one at the moment so think its a good idea to move the entire types file to be under src same as the constants.